packages/create-nuxt3-app/templates/sidebase-main/README.md

```diff
@@ -0,0 +1,231 @@
+![sidebase-logo-for-dark](.github/sidebase_logo_dark_icon_font_for_dark_bg_long.svg#gh-dark-mode-only#gh-dark-mode-only)
+![sidebase-logo-for-light](.github/sidebase_logo_light_icon_font_white_bg_long.svg#gh-light-mode-only)
```

Comment:
Can we already make this less sidebase-y?

I would:
- remove most of the sidebase references
- then remove all assets that become irelevant (e.g. the .github/ pngs and svg that make this package big)
- link to the original sidebase for further documentation
- style this more like a scaffold / starter readme for a project